<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameInfo</class>
 <widget class="QWidget" name="GameInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="picture">
                   <property name="minimumSize">
                    <size>
                     <width>320</width>
                     <height>320</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="maximumSize">
                        <size>
                         <width>40</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>标题：</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="title">
                       <property name="text">
                        <string>标题</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QLabel" name="icon_1">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>60</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="icon_2">
                       <property name="minimumSize">
                        <size>
                         <width>60</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>60</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="icon_3">
                       <property name="minimumSize">
                        <size>
                         <width>60</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>60</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="icon_4">
                       <property name="minimumSize">
                        <size>
                         <width>60</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>60</width>
                         <height>60</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_2">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="maximumSize">
                        <size>
                         <width>40</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>描述：</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPlainTextEdit" name="description">
                       <property name="styleSheet">
                        <string notr="true">QPlainTextEdit {background-color:transparent;}</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLabel" name="updateTick">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>80</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>80</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="androidButton">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>复制安卓下载地址</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="iosButton">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>40</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>复制IOS下载地址</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QListWidget" name="epsListWidget">
               <property name="styleSheet">
                <string notr="true">QListWidget {background-color:transparent;}
</string>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideRight</enum>
               </property>
               <property name="verticalScrollMode">
                <enum>QAbstractItemView::ScrollPerPixel</enum>
               </property>
               <property name="horizontalScrollMode">
                <enum>QAbstractItemView::ScrollPerPixel</enum>
               </property>
               <property name="spacing">
                <number>6</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QtCommentList" name="commentWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCommentList</class>
   <extends>QWidget</extends>
   <header location="global">.qt_comment_list.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>iosButton</sender>
   <signal>clicked()</signal>
   <receiver>GameInfo</receiver>
   <slot>CopyIos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>923</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>androidButton</sender>
   <signal>clicked()</signal>
   <receiver>GameInfo</receiver>
   <slot>CopyAndroid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddDownload()</slot>
  <slot>AddFavorite()</slot>
  <slot>StartRead()</slot>
  <slot>CopyIos()</slot>
  <slot>CopyAndroid()</slot>
 </slots>
</ui>
